.gitignore

```diff
@@ -35,4 +37,7 @@ Makefile*
 *.qmlproject.user
 *.qmlproject.user.*
 
+# vim swap files
+.*.swp
```

Comment:
Agreed, the problem was I didn't know you could have a global .gitignore until now. :sweat_smile: Thanks!